tasks/server.coffee

```diff
@@ -28,6 +29,7 @@ module.exports = (grunt) ->
     apiProxyPrefix  = grunt.config.get("server.apiProxy.prefix") || undefined
     apiProxyHost = grunt.config.get("server.apiProxy.host") || "localhost"
     apiProxyChangeOrigin = grunt.config.get("server.apiProxy.changeOrigin")
+    apiProxyUseSsl = process.env.API_USE_SSL || grunt.config.get("server.apiProxy.useSsl") || false
```

Comment:
Do we need to make this setting exposed via env var? The `WEB_PORT` setting is exposed because it's common to want to change that setting on different invocations. SSL, on the other hand, strikes me as something that would be set for the project and then never touched. My vote is to remove the env-var unless we would want to change the SSL setting per invocation (vs per project).